pkg/model/postgresql_deployment.go

```diff
@@ -169,6 +149,35 @@ func PostgresqlDeployment(cr *v1alpha1.Keycloak) *v13.Deployment {
 			},
 		},
 	}
+
+	//If not openshift append the initcontainer part
```

Comment:
This comment should probably go away. It's quite obvious what this code does.

Reply:
Agreed, removed the comment.